src/error-handler.js

```diff
@@ -0,0 +1,7 @@
+function errorHandler(err, config) {
+    console.error(err);
+    require('./remove-search-spec')(null, config);
+    require('./remove-search-json')(null, config);
+}
```

Comment:
Would it be appropriate to call `process.exit(1)`, here? Or are you exiting the process somewhere else?

Reply:
I suppose it couldn't hurt to do that here.